django_graph_api/tests/test_introspection.py

```diff
@@ -1,14 +1,268 @@
+    type_object = TypeObject(None, TypeKindEnum, None)
+    assert type_object.get_inputFields() is None
+
+
+def test_type__enum__get_interfaces():
+    type_object = TypeObject(None, TypeKindEnum, None)
+    assert type_object.get_interfaces() is None
+
+
+def test_type__enum__get_possibleTypes():
+    type_object = TypeObject(None, TypeKindEnum, None)
+    assert type_object.get_possibleTypes() is None
+
+
+def test_type__enum__get_enumValues():
+    type_object = TypeObject(None, TypeKindEnum, None)
+    assert type_object.get_enumValues() == TypeKindEnum.values
+
+
+def test_type__enum__get_ofType():
+    type_object = TypeObject(None, TypeKindEnum, None)
+    assert type_object.get_ofType() is None
+
+
+def test_type__list__get_name():
+    type_object = TypeObject(None, List(Character), None)
+    assert type_object.get_name() is None
+
+
+def test_type__list__get_fields():
+    type_object = TypeObject(None, List(Character), None)
+    assert type_object.get_fields() is None
+
+
+def test_type__list__get_inputFields():
+    type_object = TypeObject(None, List(Character), None)
+    assert type_object.get_inputFields() is None
+
+
+def test_type__list__get_interfaces():
+    type_object = TypeObject(None, List(Character), None)
+    assert type_object.get_interfaces() is None
+
+
+def test_type__list__get_possibleTypes():
+    type_object = TypeObject(None, List(Character), None)
+    assert type_object.get_possibleTypes() is None
+
+
+def test_type__list__get_enumValues():
+    type_object = TypeObject(None, List(Character), None)
+    assert type_object.get_enumValues() is None
+
+
+def test_type__list__get_ofType():
+    type_object = TypeObject(None, List(Character), None)
+    assert type_object.get_ofType() is Character
+
+
+def test_field__get_type():
+    field_object = FieldObject(None, (
+        'name',
+        CharField(),
+    ), None)
+    assert field_object.get_type() == String
+    field_object = FieldObject(None, (
+        'characters',
+        RelatedField('self'),
+    ), None)
+    field_object.data[1]._self_object_type = Character
+    assert field_object.get_type() == Character
+    field_object = FieldObject(None, (
+        'characters',
+        RelatedField(Character),
+    ), None)
+    assert field_object.get_type() == Character
+    field_object = FieldObject(None, (
+        'characters',
+        ManyRelatedField(Character),
+    ), None)
+    assert field_object.get_type() == List(Character)
+
+
+def test_field__get_args():
+    field_object = FieldObject(None, (
+        'characters',
+        ManyRelatedField(Character, types=List(String)),
+    ), None)
+    assert field_object.get_args() == (('types', List(String)),)
+    field_object = FieldObject(None, (
+        'characters',
+        ManyRelatedField(Episode, types=Int()),
+    ), None)
+    assert field_object.get_args() == (('types', Int()),)
+
+
+def test_inputvalue__get_name():
+    inputvalue_object = InputValueObject(None, (
```

Comment:
Does it make sense to add an InputObject to the test schema? Since we don't support mutations yet, maybe it doesn't make sense. But it's cool that this stuff works for that type 👍

Reply:
Input values are also used for arguments, so we do need them now.